(wduncker@LAPTOP-AFVENG10 C:Userswimdu.wslconfig)

```diff
@@ -0,0 +1,2 @@
+[wsl2]
+memory=10GB
```

Comment:
This looks like an accidental commit. Can you delete, commit that, then reinstate and add to the .gitignore?